WordPress/Classes/ViewRelated/QR Login/View Controllers/QRLoginVerifyAuthorizationViewController.swift

```diff
@@ -196,9 +196,21 @@ extension QRLoginVerifyAuthorizationViewController {
 
         enum completed {
             static let imageName = "domains-success"
-            static let title = NSLocalizedString("You're logged in!", comment: "Title for the success view when the user has successfully logged in")
-            static let subtitle = NSLocalizedString("Tap dismiss and head back to your web browser to continue.", comment: "Subtitle instructing the user to tap the dismiss button to leave the log in flow")
-            static let confirmButton = NSLocalizedString("Dismiss", comment: "Button label that dismisses the qr log in flow and returns the user back to the previous screen")
+            static let title = NSLocalizedString(
+                "qrLoginVerifyAuthorization.completedInstructions.title",
+                value: "You're logged in!",
+                comment: "Title for the success view when the user has successfully logged in"
+            )
+            static let subtitle = NSLocalizedString(
+                "qrLoginVerifyAuthorization.completedInstructions.subtitle",
+                value: "Tap dismiss and head back to your web browser to continue.",
+                comment: "Subtitle instructing the user to tap the dismiss button to leave the log in flow"
+            )
+            static let confirmButton = NSLocalizedString(
+                "qrLoginVerifyAuthorization.completedInstructions.dismiss",
+                value: "Dismiss",
+                comment: "Button label that dismisses the qr log in flow and returns the user back to the previous screen"
+            )
```

Comment:
We could be smarter here and interpolate the localized "dismiss" across both strings.